src/Security/Authentication.WsFederation/samples/WsFedSample/WsFedSample.csproj

```
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFrameworks>net461;netcoreapp2.0</TargetFrameworks>
```

```suggestion
    <TargetFrameworks>net461;netcoreapp2.1</TargetFrameworks>
```

Might have originally been 2.0, but let's cleanup as long as we're here :)